src/Component/CovidGraph/CovidGraph.jsx

```diff
@@ -1,77 +1,52 @@
-import React, { useState, useEffect } from "react";
-import axios from "axios";
+import React from "react";
+import { useContext } from "react";
+import { createContextApi } from "../ContextApi/ContextApi";
+import "../CovidGraph/CovidGraph.css";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```

Comment:
First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.